### ISSUE TITLE
Request for help

### ISSUE THREAD

**Issue by satishgc227** [foranonymous123/CNNforFaultInterpretation#7]
Respected sir/madam,
                    I wanted model for segy data.Is it possible for segy data .Please help me in this regard.

Regards,
Satish

**Comment by kelreeeeey** [comment]
Hi, 
as far as i understand, yes it is possible only if you converted your segy data to npy data type. you can see more information on segy data handling, from there i believe you can find the way to convert segy to npy/npz

**Comment by anyuzoey** [comment]
Yes, @kelreeeeey is corrent, deep learning model is only possible for numpy type of matrix. We have to either convert segy to numpy or segy to image to numpy. I hope this answered your question.
